.github/workflows/ci.yml

```diff
@@ -67,5 +67,8 @@ jobs:
       - name: Build typescript
         run: yarn lint:ci
 
+      - name: Check commit message is valid
```

Comment:
Was that removed on purpose?

The purpose of this check isn't fully completed, but it's intended to work along https://github.com/release-it/conventional-changelog#infile to generate/public automatically to npm and create the release changelog. Similar to [this](https://github.com/react-navigation/react-navigation/releases/tag/%40react-navigation%2Fstack%406.0.0-next.11)